site/docs/how-tos/operator.md

```diff
@@ -0,0 +1,305 @@
+---
+title: How to install Hazelcast Jet Operator 
+description: Step-by-step guide on how to install Hazelcast Jet Operator on your Kubernetes Cluster.
+---
+
+The [Operator Framework](https://github.com/operator-framework) is an
+open source toolkit to manage Kubernetes native applications, called
+operators, in an effective, automated, and scalable way.
+
+Hazelcast Jet provides an Operator to simplify deployment on Kubernetes
+clusters. This is a step-by-step guide how to deploy Hazelcast Jet
+cluster together with Management Center on your Kubernetes cluster.
+
+## Prerequisites
+
+- Kubernetes cluster (with admin rights) and the `kubectl` command
+  configured (you may use [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/)
+  )
+
+## Installation
+
+Hazelcast Jet Operator is published on [OperatorHub.io](https://operatorhub.io/)
+, registry for Kubernetes Operators. It can be either installed via
+Operator Lifecycle Manager or manually.
+
+### Installing with Operator Lifecycle Manager(OLM)
+
+Operator Lifecycle Manager helps you to install, update, and generally
+manage the lifecycle of all of the operators (and their associated
+services) running across your clusters.
+
+#### Installing the OLM
+
+You need to install the OLM to your cluster if it's not yet installed.
+
+To install the OLM run the commands below:
+
+```bash
+kubectl apply --validate=false -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.14.1/crds.yaml
```

Comment:
I'd try to avoid giving the commands to install OLM and especially the version. It is very quickly outdated and updating this would require manual verification steps, which makes it very hard to maintain.